nodebuilder/header/config.go

```diff
@@ -31,6 +31,9 @@ type Config struct {
 
 	Server p2p_exchange.ServerParameters
 	Client p2p_exchange.ClientParameters `toml:",omitempty"`
+
+	// enables metrics for the entire header module
+	MetricsEnabled bool
```

Comment:
I would approve instantly if we do global func

Reply:
global func or var?

Reply:
changed to var

Reply:
https://github.com/celestiaorg/celestia-node/pull/2844/commits/1bda27f371774ef1da20b79f25deeaa1e2ab3d5c

![image](https://github.com/celestiaorg/celestia-node/assets/41963722/ae2cee6e-2a2c-472a-a105-5fd1bdf8dab1)